styles.css

```diff
@@ -306,3 +306,99 @@ h2 {
   background-color: #dfe1e6;
   width: 94%;
 }
+
+.content-1 {
+  background-color: #6070ff;
+  background-image: url(image/Contact1.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 779px;
+  width: 375px;
+  border-top-left-radius: 100px;
+  padding: 8px;
+}
+
+.content-h2 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  margin-left: 40px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  margin-top: 80px;
+}
+
+.contact-me {
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #ebebff;
+  margin-left: 40px;
+  margin-right: 20px;
+}
+
+form {
+  display: flex;
+  margin-left: 12px;
+  margin-right: 20px;
+  margin-bottom: 500px;
+}
+
+input,
+textarea {
+  width: 327px;
+  height: 48px;
+  left: 16px;
+  right: 38px;
+  top: calc(50% - 18px / 2);
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+}
+
+textarea {
+  width: 327px;
+  height: 170px;
+  border: 1px solid #cfd8dc;
+  color: #b3bac5;
+  color: #000;
+}
+
+#name {
+  padding-left: 15px;
+}
+
+#mail {
+  padding-left: 15px;
+}
+
+#msg {
+  padding-left: 15px;
+}
+
+.in-touch {
+  align-items: center;
+  padding: 12px 16px;
+  width: 142px;
+  height: 48px;
+  background: #fff;
+  border-radius: 8px;
+}
+
+.in-touch:hover {
+  background-color: #6070ff;
+  color: #fff;
+  border-color: #fff;
+  cursor: pointer;
+}
```

Comment:
[OPTIONAL] Kindly follow the design guideline on Figma for all the styling values. And fix your Hover and Active(Pressed) state of your buttons 👍 🔘 
<img width="484" alt="Screenshot 2023-04-04 at 8 51 24 PM" src="https://user-images.githubusercontent.com/25887253/229879311-0add0ef3-6a55-467e-bdb1-7dee9a96d954.png">